CMakeLists.txt

```diff
@@ -46,6 +46,15 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     set(J2V8_LIB_PLATFORM_NAME "linux")
     set(J2V8_LIB_PREFIX "")
     set(J2V8_LIB_ARCH_NAME "x86")
+
+    # TODO: This hack could be used to have a different lib filename for alpine, e.g. libj2v8_alpine_x86_64
```

Comment:
I agree, the platform + architecture combination should be the primary indicator about the binary compatibility for a shared library. Since the lib for alpine would not be compatible with "default" linux distros, it should have its own identifier here and should also be handled by the java LibraryLoader.